orbit/cmd/orbit/orbit.go

```diff
@@ -491,27 +491,35 @@ func main() {
 		if err != nil {
 			return fmt.Errorf("error new orbit client: %w", err)
 		}
-		orbitNodeKey, err := getOrbitNodeKeyOrEnroll(orbitClient, c.String("root-dir"))
-		if err != nil {
-			return fmt.Errorf("error enroll: %w", err)
-		}
 
-		const orbitFlagsUpdateInterval = 30 * time.Second
-		flagRunner, err := update.NewFlagRunner(orbitClient, update.FlagUpdateOptions{
-			CheckInterval: orbitFlagsUpdateInterval,
-			RootDir:       c.String("root-dir"),
-			OrbitNodeKey:  orbitNodeKey,
-		})
-		if err != nil {
-			return err
+		if err := orbitClient.Ping(); err != nil {
```

Comment:
Let's add a comment that we use the Ping call here to receive the capabilities in the header. (Otherwise it's not obvious.)

Alternative is have the `Ping` method return the headers.

Reply:
I started with `Ping` returning a `fleet.CapabilityMap`, but I realized this might introduce bugs down the line, as you have to assign that map to a variable that might be reused later and contain stale information.

I went with the comment for now, but I _think_ this is something we can safely change in the future without fear of backwards compat